numba/cuda/device_init.py

```diff
@@ -31,6 +33,10 @@
 
 reduce = Reduce = reduction.Reduce
 
+# Expose vector types constructors as module level attributes.
+for vector_type_stub in _vector_type_stubs:
```

Comment:
Should we `del` `vector_type_stub` and maybe `_vector_type_stubs` after this loop so they don't appear in the module globals afterwards?